SharedUtils/src/main/java/com/braintreepayments/api/sharedutils/UpgradeRequiredException.kt

```diff
@@ -0,0 +1,7 @@
+package com.braintreepayments.api.sharedutils
+
+/**
+ * Exception thrown when a 426 HTTP_UPGRADE_REQUIRED response is encountered. Indicates that the
+ * API used or current SDK version is deprecated and must be updated.
+ */
+class UpgradeRequiredException(message: String) : Exception(message)
```

Comment:
😅 I've never seen this one before, interesting we have a custom exception for it.

Reply:
I haven't seen this one either! It appears to be thrown when the API returns a 426 status code.